ext/ClimaOceanSpeedyWeatherExt/speedy_weather_exchanger.jl

```diff
@@ -95,16 +92,33 @@ function interpolate_atmosphere_state!(interfaces, atmos::SpeedySimulation, coup
     Qse = exchanger.cpu_surface_state.Qs
     Qle = exchanger.cpu_surface_state.Qℓ
 
-    ConservativeRegridding.regrid!(ua,  ue,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(va,  ve,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(Ta,  Te,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(qa,  qe,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(pa,  pe,  regridder, exchange_areas)
-    ConservativeRegridding.regrid!(Qsa, Qse, regridder, exchange_areas)
-    ConservativeRegridding.regrid!(Qla, Qle, regridder, exchange_areas)
+    ConservativeRegridding.regrid!(ue,  regridder, ua)
+    ConservativeRegridding.regrid!(ve,  regridder, va)
+    ConservativeRegridding.regrid!(Te,  regridder, Ta)
+    ConservativeRegridding.regrid!(qe,  regridder, qa)
+    ConservativeRegridding.regrid!(pe,  regridder, pa)
+    ConservativeRegridding.regrid!(Qse, regridder, Qsa)
+    ConservativeRegridding.regrid!(Qle, regridder, Qla)
     
     arch = architecture(exchange_grid)
     fill_exchange_fields!(arch, exchange_state, exchanger.cpu_surface_state)
 
     return nothing
-end
\ No newline at end of file
+end
+
+function compute_net_atmosphere_fluxes!(coupled_model::SpeedyCoupledModel)
+    atmos = coupled_model.atmosphere
+
+    # All the location of these fluxes will change
+    Qs = similarity_theory_fields.sensible_heat
```

Comment:
We need to come up with a notation guide, but elsewhere we define `Qs` as "downwelling shortwave" and `Qc` as sensible.

We can def change whatever but should be aware and try to stay consistent.